tools/logger/convert.c

```diff
@@ -103,6 +103,7 @@ static const char * get_component_name(uint32_t component_id) {
 		CASE(CHMAP);
 		CASE(ASRC);
 		CASE(NOTIFIER);
+		CASE(PROBE);
```

Comment:
I think it's also needed in testbench or host/lib

Reply:
added